src/strategies/landdao-token-tiers/examples.json

```diff
@@ -0,0 +1,8914 @@
+          "8753": 2,
+          "8754": 3,
+          "8755": 2,
+          "8756": 2,
+          "8757": 3,
+          "8758": 1,
+          "8759": 1,
+          "8760": 3,
+          "8761": 1,
+          "8762": 1,
+          "8763": 3,
+          "8764": 3,
+          "8765": 3,
+          "8766": 1,
+          "8767": 2,
+          "8768": 2,
+          "8769": 1,
+          "8770": 1,
+          "8771": 2,
+          "8772": 3,
+          "8773": 1,
+          "8774": 2,
+          "8775": 3,
+          "8776": 1,
+          "8777": 3,
+          "8778": 3,
+          "8779": 1,
+          "8780": 3,
+          "8781": 2,
+          "8782": 2,
+          "8783": 3,
+          "8784": 2,
+          "8785": 2,
+          "8786": 1,
+          "8787": 2,
+          "8788": 1,
+          "8789": 1,
+          "8790": 1,
+          "8791": 2,
+          "8792": 2,
+          "8793": 3,
+          "8794": 1,
+          "8795": 1,
+          "8796": 1,
+          "8797": 3,
+          "8798": 2,
+          "8799": 3,
+          "8800": 1,
+          "8801": 2,
+          "8802": 3,
+          "8803": 1,
+          "8804": 3,
+          "8805": 3,
+          "8806": 1,
+          "8807": 1,
+          "8808": 2,
+          "8809": 1,
+          "8810": 3,
+          "8811": 1,
+          "8812": 2,
+          "8813": 3,
+          "8814": 1,
+          "8815": 1,
+          "8816": 2,
+          "8817": 1,
+          "8818": 2,
+          "8819": 2,
+          "8820": 3,
+          "8821": 3,
+          "8822": 1,
+          "8823": 1,
+          "8824": 2,
+          "8825": 1,
+          "8826": 3,
+          "8827": 3,
+          "8828": 1,
+          "8829": 3,
+          "8830": 2,
+          "8831": 1,
+          "8832": 1,
+          "8833": 3,
+          "8834": 1,
+          "8835": 3,
+          "8836": 2,
+          "8837": 3,
+          "8838": 3,
+          "8839": 3,
+          "8840": 2,
+          "8841": 1,
+          "8842": 3,
+          "8843": 1,
+          "8844": 1,
+          "8845": 2,
+          "8846": 3,
+          "8847": 3,
+          "8848": 3,
+          "8849": 2,
+          "8850": 1,
+          "8851": 2,
+          "8852": 1,
+          "8853": 3,
+          "8854": 3,
+          "8855": 1,
+          "8856": 2,
+          "8857": 3,
+          "8858": 3,
+          "8859": 2,
+          "8860": 1,
+          "8861": 1,
+          "8862": 2,
+          "8863": 3,
+          "8864": 1,
+          "8865": 3,
+          "8866": 3,
+          "8867": 1,
+          "8868": 3,
+          "8869": 3,
+          "8870": 2,
+          "8871": 3,
+          "8872": 2,
+          "8873": 2,
+          "8874": 2,
+          "8875": 3,
+          "8876": 1,
+          "8877": 1,
+          "8878": 2,
+          "8879": 2,
+          "8880": 2,
+          "8881": 2,
+          "8882": 3,
+          "8883": 3,
+          "8884": 1,
+          "8885": 1,
+          "8886": 1,
+          "8887": 1,
+          "8888": 1
```

Comment:
Hi @ethantddavis Adding tokens like this can potentially create scalability issues on the snapshot server, is it possible to fetch this list from somewhere? maybe from an IPFS file?

Reply:
Thanks for the feedback @ChaituVR, I will set up an IPFS file to store the data. I see the similarity to tokenId-range-weights, but many token weights for our use case are not within a range, instead were assigned randomly. I am not sure how I would separate common code in this scenario.